gravity.sh

```diff
@@ -52,6 +52,20 @@ else
   exit 1
 fi
 
+# Set up tmp dir in case it's not configured
+if [ -z "${GRAVITY_TMPDIR}" ]; then
+  GRAVITY_TMPDIR="/tmp"
+else
+  if [ ! -e "${GRAVITY_TMPDIR}" ]; then
+    echo -e "  ${COL_LIGHT_RED}Gravity temporary directory does not exist, falling back to /tmp. ${COL_NC}"
+    GRAVITY_TMPDIR="/tmp"
+  fi
```

Comment:
Not sure if there's a need to check for both existence and if is a directory. Should be able to simplify to just checking if the path is a directory. `-d` checks both existence and directory-ness. 

Might want to add in `-w` to check if the directory is writeable as well.

Reply:
Let's keep it simple indeed, updated the code accordingly.